From 9068c5abeb7ba920a55b1a76ff465440d313ef42 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Wed, 13 Oct 2021 15:10:17 +0200
Subject: [PATCH] Fixes for backporting errors

After refactoring the old patches to the new kernel
some errors occured during compiling.
For the moment fix those until fixing the initial parent
patches

Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io>
---
 arch/arm64/boot/dts/compulab/cl-som-imx8.dts | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
index da3b26d70407..33e4337db4a2 100644
--- a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
+++ b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
@@ -237,8 +237,8 @@
 		touchscreen-inverted-y = "true";
 
 		esd-recovery-timeout-ms = <2000>;
-		interrupts-extended = GPIRQ_GT911;
-		irq-gpios = GP_GT911_IRQ;
+		interrupts-extended = <&gpio4 26 IRQ_TYPE_LEVEL_HIGH>;
+		irq-gpios = <&gpio4 26 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
 
@@ -492,16 +492,8 @@
 			>;
 		};
 
-		pinctrl_pwm1: pwm1grp {
-			fsl,pins = <
-				MX8MQ_IOMUXC_SPDIF_EXT_CLK_PWM1_OUT     0x16
-			>;
-		};
-
 		pinctrl_i2c4_gt911: i2c4-gt911grp {
 			fsl,pins = <
-				#define GPIRQ_GT911		<&gpio4 26 IRQ_TYPE_LEVEL_HIGH>
-				#define GP_GT911_IRQ		<&gpio4 26 GPIO_ACTIVE_HIGH>
 				MX8MQ_IOMUXC_SAI2_TXD0_GPIO4_IO26	0x1D6
 			>;
 		};
-- 
2.17.1

